.github/integ-config/integ-all.yml

```diff
@@ -7,39 +7,68 @@ extended_browser_list: &extended_browser_list
   - firefox
   - edge
 
-datastore_auth_scenarios_sample_spec: &datastore_auth_scenarios_sample_spec
-  - owner-based-default owner-based-default # TODO: Add `owner-based-operations` when tests pass
-  - static-user-pool-groups-default static-user-pool-groups-default
-  - static-user-pool-groups-operations static-user-pool-groups-operations
-  - owner-and-group-different-models-default owner-and-group-different-models-default
-  - owner-and-group-same-model-default owner-and-group-same-model-default
-  - owner-and-group-same-model-operations owner-and-group-same-model-operations
-  - dynamic-user-pool-groups-default dynamic-user-pool-groups-default
-  # - dynamic-user-pool-groups-owner-based-default dynamic-user-pool-groups-owner-based-default
-  - private-auth-default private-auth-default
-  - private-auth-iam private-auth-iam
-  - public-auth-default public-auth-default
-  - public-auth-iam public-auth-iam
-  - owner-custom-claim-default owner-custom-claim-default
-  - owner-custom-field-default owner-custom-field-default
-
-datastore_auth_scenarios_sample_spec_v2:
-  &datastore_auth_scenarios_sample_spec_v2
-  - v2/owner-based-default-v2 owner-based-default # TODO: Add `owner-based-operations` when tests pass
-  - v2/static-user-pool-groups-default-v2 static-user-pool-groups-default
-  - v2/static-user-pool-groups-operations-v2 static-user-pool-groups-operations
-  - v2/owner-and-group-different-models-default-v2 owner-and-group-different-models-default
-  - v2/owner-and-group-same-model-default-v2 owner-and-group-same-model-default
-  - v2/owner-and-group-same-model-operations-v2 owner-and-group-same-model-operations
-  - v2/private-auth-iam-v2 private-auth-iam
-  - v2/public-auth-default-v2 public-auth-default
-  - v2/public-auth-iam-v2 public-auth-iam
-  - v2/owner-custom-claim-default-v2 owner-custom-claim-default
-  - v2/owner-custom-field-default-v2 owner-custom-field-default
-  - v2/dynamic-user-pool-groups-default-v2 dynamic-user-pool-groups-default
-  - v2/private-auth-default-v2 private-auth-default
+datastore_auth_scenarios: &datastore_auth_scenarios
+  # TODO: Add `owner-based-operations` when tests pass
+  - sample_name: owner-based-default #39
```

Comment:
The numbers at the end indicate the total character length of sample_name+spec.
The current limit is 85. Anything above 85 fails the test, hence the commented test at the bottom

will cleanup these numbers after initial discussion